File: sdk/.net/ardvro.sdk.net/component/sqljson/db/DbQueryAggregate.cs
namespace ardvro.sdk.net.component.sqljson.db;

using ardvro.core.ext.connection;

[Serializable]
public class DbQueryAggregate : DbQuery
{
    public string Field = "*";

    public string AggregateType;

    public DbQueryAggregate(DbQuery query, String field, String aggregateType, IConnection connection)
        :base(connection)
    {
        Base = query;
        Connection = connection;
        QueryType = "AGGREGATE";
        Field = field;
        AggregateType = aggregateType;
    }



}

File: sdk/android/sqljson/DbQueryAggregate.java
package ardvro.component.sqljson;

import java.io.Serializable;

import ardvro.core.connection.IConnection;

public class DbQueryAggregate extends DbQuery implements Serializable
{
    public String Field = "*";

    public String AggregateType;

    public DbQueryAggregate(DbQuery query, String field, String aggregateType, IConnection connection)
    {
        super(connection);
        Base = query;
        Connection = connection;
        QueryType = "AGGREGATE";
        Field = field;
        AggregateType = aggregateType;
    }



}
